doc/qbk/4.0.containers.qbk

```diff
@@ -0,0 +1,41 @@
+[/
+    Copyright (c) 2022 Alan de Freitas (alandefreitas@gmail.com)
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+    Official repository: https://github.com/CPPAlliance/url
+]
+
+[section:containers Containers]
+
+The library provides containers for representing URLs according to
+their generic syntax for well known schemes. The three main
+URL containers are:
+
+* __url_view__: A non-owning reference to a valid URL.
+* __url__: A modifiable container for a URL.
+* __static_url__: A URL with fixed-capacity storage.
+
+These containers are meant to cover common use cases such
```

Comment:
This language could be a little more formal ("containers are meant" sounds a bit soft)